src/Rule/Trait/LimitHandlerTrait.php

```diff
@@ -15,54 +14,26 @@ trait LimitHandlerTrait
     /**
      * Runs limits specific validation.
      *
-     * @param mixed $value A validated value in its raw form.
      * @param object $rule A rule matching to this handler.
-     * @param ValidationContext $context Validation context.
      * @param int $number A validated number to compare with set limits.
      * @param Result $result Result for adding errors.
      *
      * @see LimitTrait for information about limits and messages.
      */
-    private function validateLimits(
-        mixed $value,
-        object $rule,
-        ValidationContext $context,
-        int $number,
-        Result $result
-    ): void {
+    private function validateLimits(object $rule, int $number, Result $result): void
```

Comment:
I think we should save removed parameters (min, max, exactly) for messages. They can be useful for custom messages.

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006452567

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006471638